src/mscorlib/shared/System/Memory.cs

```diff
@@ -276,6 +273,7 @@ public Span<T> Span
                 if (_index < 0)
                 {
                     Debug.Assert(_length >= 0);
+                    Debug.Assert(_object != null);
```

Comment:
The `if (_object != null)` case we discussed will be fixed in a separate PR?

Reply:
Oh, wait, nevermind.

Reply:
> The `if (_object != null)` case we discussed will be fixed in a separate PR?

Pin looks correct to me, so there was nothing to update.